requirements/dev.txt

```diff
@@ -2,6 +2,6 @@
 -r docs.txt
 -r flake8.txt
 
-django-debug-toolbar==1.3.2
```

Comment:
If you think the DjDT should go for good, then this needs to go to (not be reverted back to install an old version)